go/dpagg/mean.go

```diff
@@ -213,6 +216,102 @@ func (bm *BoundedMeanFloat64) Result() float64 {
 	return clamped
 }
 
+// ComputeConfidenceInterval computes a confidence interval that contains the true mean with a probability
+// greater than or equal to 1 - alpha using the confidence interval for sum and count.
```

Comment:
We added an extra couple lines here to BoundedSum and Count due to a comment Dennis had, could you please add those here as well?